### ISSUE TITLE
Update nav names

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4845]
[placeholder ticket]

Names can be updated in the CMS but we will need to redirect all the pages.

Consider localization.

Blocked by Treejack test https://github.com/mozilla/foundation.mozilla.org/issues/4649

**Comment by phildexter** [comment]
This is more complicated as we'll need to handle redirects - another ticket will handle this work.
